src/sass/_Fabric.Typography.Mixins.scss

```diff
@@ -5,7 +5,7 @@
 // --------------------------------------------------
 // Fabric Core Typography mixins
 
-@mixin ms-baseFont() {
+@mixin ms-baseFont {
   font-family: $ms-font-family-base;
```

Comment:
Thanks @mikewheaton. Like I said, I think it's absolutely the right thing to add, and you're absolutely right--this will accrue even greater benefits to other aspects of the project. My concern is that since this will be potentially broadly-impacting, I want to be sure we're careful about which properties we add when and that the appropriate consumers are alerted to the change and can plan accordingly.